src/store/epics/helpers/urls.js

```diff
@@ -0,0 +1,14 @@
+
+let hostname;
+if (typeof window.location !== "undefined") {
```

Comment:
`window` is defined on mobile, but not `window.location`?

Reply:
I'd think that `window` wouldn't be defined outside the browser.

Reply:
`window` is defined in a react native environment. It's possible we end up with a different mobile scenario (say, not CapacitorJS) and will have to tweak this.

Reply:
Good to know, thanks